src/app/components/apparatus-entry/apparatus-entry.component.ts

```diff
@@ -12,22 +18,49 @@ import { ApparatusEntryDetailComponent } from './apparatus-entry-detail/apparatu
 })
 @register(ApparatusEntry)
 export class ApparatusEntryComponent {
+  highlightData$ = new BehaviorSubject<HighlightData | undefined>({
+    highlight: true,
+    highlightColor: AppConfig.evtSettings.edition.readingColorLight,
+  });
+
+  constructor(
+    private evtModelService: EVTModelService,
+    @Optional() private parentDetailComponent?: ApparatusEntryDetailComponent,
+    @Optional() @SkipSelf() private parentAppComponent?: ApparatusEntryComponent,
+  ) {
+    this.isInsideAppDetail = !!this.parentDetailComponent;
+    this.appIsInsideApp = !!this.parentAppComponent;
+  }
   @Input() data: ApparatusEntry;
 
   public opened = false;
   public isInsideAppDetail: boolean;
+  public appIsInsideApp: boolean;
   public nestedApps: ApparatusEntry[] = [];
 
   variance$ = this.evtModelService.appVariance$.pipe(
     map((variances) => variances[this.data.id]),
     shareReplay(1),
   );
 
-  constructor(
-    private evtModelService: EVTModelService,
-    @Optional() private parentDetailComponent?: ApparatusEntryDetailComponent,
-  ) {
-    this.isInsideAppDetail = this.parentDetailComponent ? true : false;
+  @HostListener('mouseenter') onMouseEnter() {
+    if (this.appIsInsideApp) {
+      this.parentAppComponent.highlightData$.next({
+        highlight: true,
+        highlightColor: AppConfig.evtSettings.edition.readingColorLight,
+      });
+    }
+    this.highlightData$.next({
+      highlight: true,
+      highlightColor: AppConfig.evtSettings.edition.readingColorDark,
+    });
+  }
+
+  @HostListener('mouseleave') onMouseLeave() {
+    this.highlightData$.next({
+      highlight: true,
```

Comment:
Shouldn't this be set to `false`?

Reply:
No, if I set it to false the highlight disappears

Reply:
When should this be set to false?

OnMouse enter it is set to true, and also OnMouseLeave